spread.yaml

```diff
@@ -3,8 +3,8 @@ project: snapd
 environment:
     REUSE_PROJECT: $(echo $REUSE_PROJECT)
     PROJECT_PATH: /gopath/src/github.com/snapcore/snapd
```

Comment:
We also need to use `$PROJECT_PATH` in https://github.com/snapcore/snapd/blob/master/tests/main/gccgo/task.yaml#L13

Thanks!